### DISCUSSION TITLE
Welcome to JMusicBot Discussions!

### DISCUSSION THREAD

**Discussion by jagrosh** [jagrosh/MusicBot#600] [General]
Welcome to the JMusicBot Discussions. Here's where you can discuss JMusicBot, brainstorm new features, or ask for help from the community. 

### Resources
* [Wiki](https://jmusicbot.com) - The first place to look for instructions
* [Issues](https://github.com/jagrosh/MusicBot/issues) - Report reproducible bugs or request thought-out features
* [Discord](https://discord.gg/0p9LSGoRLu6Pet0k) - Development server and a demo of the bot (Clarity)
* [Latest Release](https://github.com/jagrosh/MusicBot/releases/latest) - Latest download of JMusicBot

**Comment by anxlz** [comment]
play from spotify

**Reply by jagrosh** [reply]
Playing from Spotify is not possible, for three main reasons:
1. This would require an additional API key to look up songs on Spotify, which would be confusing for some users and overall make the bot feel more complicated to use.
2. Streaming from Spotify isn't allowed, so the bot would have to "guess" and just play the closest match from YouTube, which is unreliable at best. 
3. Discord already has native support for syncing up Spotify playlists between users, so there's no reason to not just use that (and get better audio anyway from Spotify instead of from a Discord voice channel).

**Reply by anxlz** [reply]
could you just add it with another version there a lot of ones know how to get api ,token ,id ,and secret and its easy to make it from spotify developers website and a lot of ones need to play from spotify.

**Reply by jagrosh** [reply]
No, if you look at point 2 in that list, streaming from spotify isn't possible even with an API key. Spotify will not be added; do not bring it up again.

**Comment by abkux** [comment]
Is there anyway that we can make changes in help command ?

**Comment by KarolKozlowski** [comment]
TLDR: How to update wiki?
I would like to resolve #597 - I have working setup, but this requires PR to main repo (the easy part) as well as updating the wiki, which is in a separate repo that does not support forking. Is there an established workflow for updating the wiki? Or should I just drop a markdown with the instructions along with the service file?

**Reply by jagrosh** [reply]
Specifically to the question of how to update the wiki (since you can't PR to it): the best bet would be to just open an issue containing the recommended changes.

**Reply by KarolKozlowski** [reply]
OK, will do. Thanks!

**Reply by minecraftchest1** [reply]
You can also just click the pencil in the top right and edit it that way. Everyone can edit a wiki afiak.

**Comment by marlussss** [comment]
Is there a command that i can set the lplay (game) for ever. bc when i start the bot new the bot new the bot delets his game playing. But all in all the bot is fucking insaine thanks man. Greats from Austria

**Reply by jagrosh** [reply]
You'll want to set it in the config file instead of with the command.

**Comment by Zappy52** [comment]
How do I change the client token i want to use it on another bot

**Comment by ajburch92** [comment]
Thank you for creating the bot - can someone help me with this blocking direct messages error? 
https://github.com/jagrosh/MusicBot/discussions/748

**Comment by Nikos343** [comment]
How to fix this ?
![GIGI](https://github.com/jagrosh/MusicBot/assets/165834780/4a1adb15-ebc5-4ce0-9f5e-0eb9407589fa)

**Reply by Nikos343** [reply]
It connects and then disconnects over and over again

**Comment by chillybeam90-eng** [comment]
The [2048 Cupcake Game](https://2048-cup-cakes.com/) is a unique and entertaining puzzle game that has captured the attention of players worldwide. Inspired by the original 2048 number puzzle, this version replaces numbers with colorful cupcakes, creating a more visually appealing and enjoyable experience. The combination of simple gameplay, creative design, and strategic challenges makes it a game that players can enjoy again and again.
